docker/witnet-rust/wip0010-recovery.sh

```diff
@@ -0,0 +1,156 @@
+
+# Just a pretty logging helper
+function log {
+  echo "[WIP0010_RECOVERY] $1"
+}
+
+# A helper for calculating ETAs
+function eta {
+  START=$1
+  NOW=$2
+  PROGRESS=$3
+  if [ "$PROGRESS" == "00" ]; then
+    echo "will be shown as synchronization moves forward..."
+  else
+    ELAPSED=$(( NOW - START ))
+    SPEED=$((PROGRESS * 1000 / ELAPSED))
+    if [ "$SPEED" == "0" ]; then
+        SPEED=1
+    fi
+    REMAINING_PROGRESS=$(( 10000 - PROGRESS ))
+    REMAINING_TIME=$((REMAINING_PROGRESS / SPEED / 1000))
+    echo $(( REMAINING_TIME / 60 )) minutes $((REMAINING_TIME % 60)) seconds
```

Comment:
Now it says `ETA: 0 minutes 0 seconds`, let's just revert it to how it was before :see_no_evil:

Reply:
I think I was dividing where I should be multiplying.

Reply:
`[WIP0010_RECOVERY] Still rewinding and synchronizing. Progress: 6.25%. ETA: 115 minutes 18 seconds`

Reply:
Fixed in f5dcf21